src/test/scala/edu/ie3/powerFactory2psdm/converter/types/TransformerType2WConverterSpec.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.powerFactory2psdm.converter.types
+
+import edu.ie3.powerFactory2psdm.common.ConverterTestData.getTransformer2wType
+import edu.ie3.powerFactory2psdm.exception.pf.ConversionException
+import edu.ie3.scalatest.QuantityMatchers
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+import math.abs
+
+class TransformerType2WConverterSpec
+    extends Matchers
+    with AnyWordSpecLike
+    with QuantityMatchers {
+
+  "A Transformer2wTypeConverter" should {
+    val conversionPair = getTransformer2wType("SomeTrafo2wType")
+    val input = conversionPair.input
+    val expected = conversionPair.result
+    implicit val quantityTolerance: Double = 1e-3
```

Comment:
Based on my recently gained experience, the transformer parameters can have a good influence on the power flow results. So therefore, you might be a bit more strict with the comparison (e.g. 1e-6 or 1e-9)